fastlane/lib/fastlane/helper/git_helper.rb

```diff
@@ -3,36 +3,38 @@ module Actions
     GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
 
     def self.git_log_between(pretty_format, from, to, merge_commit_filtering, date_format = nil, ancestry_path)
-      command = ['git log']
-      command << "--pretty=\"#{pretty_format}\""
-      command << "--date=\"#{date_format}\"" if date_format
+      command = %w(git log)
```

Comment:
This is equivalent to `['git', 'log']`, correct?

Reply:
If you know about it, yeah ;) Ruby sometimes is not the most discoverable language - but I am of course fine with it.

The reason for switching from `git log` to `git, log` is consistency I guess?

Reply:
Great. If you prefer to rather not have these cryptic constucts, I can also go with plain arrays, and teach rubocop to complain if they are ever used.

Consistency indeed is one of the reasons, but in this case `git log` won't work anymore: as we pass each argument separately, `shelljoin` will consider each array element a **single** argument. This is the power of `shelljoin`, but it also means that `git log` will be shell-escaped and the OS will look for a binary called `git log` (with a space inside).

Reply:
I am the worst benchmark here, as I am really dabbling in a language I haven't really mastered. And `%w(` is google-able enough that this should not be a problem for anyone really.

> but in this case git log won't work anymore:

A good point, I didn't see that.